examples/README.md

```diff
@@ -0,0 +1,47 @@
+# Celerity Example Projects
+
+These example projects are a useful starting point for developing your own Celerity
+application. All examples will also be built automatically in-tree with the runtime
+when the `CELERITY_BUILD_EXAMPLES` CMake option is set (true by default).
+
+## Setup
+
+Begin by copying an example project folder that best fits your use case,
+while resolving symlinks:
+```shell
+cp -rL celerity-runtime/examples/convolution my-project
+```
+
+## Build Configuration
+
+Configure the project with CMake and make sure Celerity can be found by setting
+[`CMAKE_PREFIX_PATH`](https://cmake.org/cmake/help/latest/variable/CMAKE_PREFIX_PATH.html)
+to include the Celerity installation directory. Depending on your SYCL implementation,
+you may also have to specify its library installation directory and target options.
+
+### With hipSYCL (exemplary)
+
+```shell
+cd my-project
+cmake -B build \
+    -DCMAKE_PREFIX_PATH="../celerity-runtime-install/lib/cmake;../hipSYCL-install/lib/cmake" \
```

Comment:
Here and below: `-DCMAKE_PREFIX_PATH="../celerity-runtime-install;../hipSYCL-install"` should suffice!

Reply:
I'll get rid of the hipSYCL line after #65 .